_integration/testsuite/install-contour-working.sh

```diff
@@ -183,5 +183,16 @@ EOF
 ${KUBECTL} apply -f ${REPO}/examples/ratelimit/02-ratelimit.yaml
 ${KUBECTL} apply -f ${REPO}/examples/ratelimit/03-ratelimit-extsvc.yaml
 
+# Create a GatewayClass
```

Comment:
Is this just not to duplicate all over the place? Currently, the tests define their own classes. For now, this is fine but will need to change in the future once we start processing different configs from the class.

Reply:
Yeah, just to avoid duplication for now. Definitely need to figure out the "different Contour installs/configs for different tests" story soon, so we can address this as part of that I think.